lib/shopify-cli/oauth.rb

```diff
@@ -76,13 +77,21 @@ def initiate_authentication(url)
       params.merge!(challange_params) if secret.nil?
       uri = URI.parse("#{url}#{auth_path}")
       uri.query = URI.encode_www_form(params.merge(options))
+      uri
+    end
+
+    def output_authentication_info(uri)
+      login_location = service == 'admin' ? 'development store' : 'Shopify Partners account'
+      ctx.puts(
+        "{{i}} Authentication Required, login to the url below with your #{login_location} credentials to continue"
```

Comment:
Some minor content nits:

```suggestion
        "{{i}} Authentication required. Login to the URL below with your #{login_location} credentials to continue."
```